### DISCUSSION TITLE
did skiovox get patched on chrome 126?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1132] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
did it get patched and if it did does anyone have a exploit that works

**Comment by AshtonDavies** [comment]
It has had multiple patches. I think the latest patch is in v125 if I am not mistaken.

**Reply by Balistic123** [reply]
if I'm correct though I have kernver 3 so I can downgrade to 120 and do skiovox?

**Reply by Balistic123** [reply]
also I have been working on finding an exploit for a new skiovox exploit, and I figured out that if you click on add new wifi then type in random stuff hit enter then click diagnose then click open settings twice it opens up page that allows you to turn on chromevox inside the app even if it is blocked do you think we could do something with that?

**Reply by Balistic123** [reply]
@AshtonDavies

**Reply by AshtonDavies** [reply]
> also I have been working on finding an exploit for a new skiovox exploit, and I figured out that if you click on add new wifi then type in random stuff hit enter then click diagnose then click open settings twice it opens up page that allows you to turn on chromevox inside the app even if it is blocked do you think we could do something with that?

The links in ChromeVox might not work anymore. Go ahead and try if you want.

**Reply by Balistic123** [reply]
they don't but would there be some way using accessibility screen to exploit, and also I'm on kernver 3 so wouldn't I be able to downgrade to 120 and use skiovox? and one more thing, where can I learn about Chromebook code so I can learn to find exploits too?

**Reply by Balistic123** [reply]
@AshtonDavies

**Reply by Balistic123** [reply]
@AshtonDavies

**Comment by SIMONHSCH** [comment]
skiovox dsnt work anymore for me because my school added policies to all kiosk apps that block all URLs when I use skiovox 😭

**Reply by Balistic123** [reply]
what version are you on?

**Reply by Balistic123** [reply]
do you know what the policy is called @SIMONHSCH ?

**Reply by SIMONHSCH** [reply]
i'm on 124 and the policy is called URLBlocklist
